setup.cfg

```diff
@@ -8,6 +8,6 @@ exclude =
 
 [tool:pytest]
 addopts = --flake8
-flake8-ignore = E133 E226 E228 N802 N803 N806
+flake8-ignore = E133 E226 E228 N802 N803 N806 E266
     # TODO: remove E266 lines when Task documentation is converted to rst in DM-14207.
```

Comment:
This comment line is relevant (I think it was copied from another and left in by mistake) once the per-file ignores are added.